doc/ansible/dev/retroactively-owned-resources.md

```diff
@@ -0,0 +1,162 @@
+# Owner References for Existing Resources
+
+Owner references are automatically injected *only during creation of
+resources*. Enabling owner reference injection *will not update objects*
+created while [owner reference injection is
+disabled](./advanced_options.md#turning-off-dependent-watches-and-owner-reference-injection)
+.
+
+This guide will demonstrate how to retroactively set owner references
+for existing resources.
+
+### Owner References and Annotations
+
+Dependent resources *within the same namespace as the owning CR* are
+tracked with the `ownerReference` field.
+
+`ownerReference` structure:
+  * apiVersion: {group}/{version}
+  * kind: {kind}
+  * name: {metadata.name}
+  * uid: {metadata.uid}
+
+**Example:**
+
+```yaml
+
+metadata:
+  ...(snip)
+  ownerReferences:
+    - apiVersion: cache.example.com/v1alpha1
+      kind: Memcached
+      name: example-memcached
+      uid: ad834522-d9a5-4841-beac-991ff3798c00
+```
+
+An `annotation` is used instead of an `ownerReference` if the dependent
+resource is in a different namespace than the CR or the dependent
+resource is a cluster level resource.
+
+`annotation` structure:
+  * operator-sdk/primary-resource: {metadata.namepace}/{metadata.name}
+  * operator-sdk/primary-resource-type: {kind}.{group}
+
+*Note: <group> must be determined by splitting the apiVersion field at the "/"*
+
+```yaml
+metadata:
+  annotations:
+    operator-sdk/primary-resource: default/example-memcached
```

Comment:
Why the default here? Is it the namespace?

Reply:
`default` is the namespace used in the Memcached example.

> Is it the namespace +"/" + name of the CR?

Yes

Reply:
WDYT about improve the doc an make it clear?

Reply:
The structure of the annotation is on lines 60/61.  Combined with the example output you suggested on line 16, do you think it will be clear to users now?

Reply:
WDYT about just a note as, for example, something as:

**NOTE:** The `operator-sdk/primary-resource` is formed by `<namespace>/<cr-name>.